includes/class-wc-payments-utils.php

```diff
@@ -177,15 +177,24 @@ public static function zero_decimal_currencies(): array {
 	 */
 	public static function supported_countries(): array {
 		return [
+			'AT' => __( 'Austria', 'woocommerce-payments' ),
 			'AU' => __( 'Australia', 'woocommerce-payments' ),
+			'BE' => __( 'Belgium', 'woocommerce-payments' ),
 			'CA' => __( 'Canada', 'woocommerce-payments' ),
+			'CH' => __( 'Switzerland', 'woocommerce-payments' ),
 			'DE' => __( 'Germany', 'woocommerce-payments' ),
 			'ES' => __( 'Spain', 'woocommerce-payments' ),
 			'FR' => __( 'France', 'woocommerce-payments' ),
 			'GB' => __( 'United Kingdom (UK)', 'woocommerce-payments' ),
+			'HK' => __( 'Hong Kong', 'woocommerce-payments' ),
 			'IE' => __( 'Ireland', 'woocommerce-payments' ),
 			'IT' => __( 'Italy', 'woocommerce-payments' ),
+			'MY' => __( 'Malaysia', 'woocommerce-payments' ),
```

Comment:
Malaysia should not be included just yet. More details: paJDYF-1nZ-p2#comment-7117 and pbUcTB-k2-p2#comment-1493

Reply:
Thanks for the ping! Reverted in def8b81cd5ccd22547a74455f34e5b9f037ae6e6